tool/tctl/common/user_command.go

```diff
@@ -55,6 +56,10 @@ type UserCommand struct {
 	allowedAzureIdentities    []string
 	allowedGCPServiceAccounts []string
 	allowedRoles              []string
+	hostUserUID               string
+	hostUserUIDProvided       bool
```

Comment:
Why do you need separate bool flags? Is the empty string for UID not enough to indicate that it wasn't provided?

Reply:
If the user wants to unset the host user UID/GID and the flag wasnt passed it would always get overwritten without this

Reply:
I see, so `--host-user-uid=""` can be used to unset it. That seems like something no one is likely to figure out on their own, and it's not mentioned in the docs or help message.

I would either update docs/help, or have a separate flag for `--unset-uid` which would be more intuitive.